src/neuroconv/tools/nwb_helpers/_configuration_models/_hdf5_dataset_io.py

```diff
@@ -75,6 +75,9 @@ def get_data_io_kwargs(self) -> Dict[str, Any]:
             elif self.compression_method is None:
                 compression_bundle = dict(compression=False)
         else:
-            compression_bundle = dict(compression=self.compression_method, compression_opts=self.compression_options)
+            compression_opts = None
+            if self.compression_options is not None:
```

Comment:
I don't understand what are we doing here. Can you provide context?

Reply:
It's copied/pasted from https://github.com/catalystneuro/neuroconv/pull/571/files#diff-708845c5b3ce542157fc2da62133d4723159b4f250d1bc7fb83d7f2c1408feacL62-L64 (so a very small amount of duplicated code, doesn't seem worth to make a separate method though), but forgot the explaining comment above

The problem is that the `compression_opts` argument of the `H5DataIO` is very particular about how it likes things passed